veps/vep-1.md

```diff
@@ -85,7 +85,7 @@ The following is not subject to the Backwards and Forwards Compatibility Promise
 
 ## Support Lifecycle
 
-Because the release cycle includes a new release every 12 weeks, most fixes will only be pushed to the master branch. An exception to this process will be made for high severity bugs, in which case a fix will be available in the weekly release for all currently supported versions.
+Because the release cycle includes a new release every 12 weeks, most fixes will only be pushed to the main branch. An exception to this process will be made for high severity bugs, in which case a fix will be available in the weekly release for all currently supported versions.
 
 High severity bugs in this context include [CVEs](https://cve.mitre.org/), data corruption, wrong results or outage inducing issues. The Vitess team **MAY** decide to only backport bugs with higher user impact.
 
```

Comment:
Can you add a new section to the bottom of both of these docs to call out "notable edits"? I was thinking along the lines of the following:

```
## Edits

- <date>
    - <author>, <high-level summary of changes> (<link to PR>)
- <date2> (and so on)
```

Reply:
Oh I missed that _two_ docs changed. So yeah all new information should be added to both.

Reply:
Look OK now?